drivers/include/disp_dev.h

```diff
@@ -133,11 +143,27 @@ uint8_t disp_dev_color_depth(disp_dev_t *dev);
 /**
  * @brief   Invert the display device colors
  *
- * @param[in] dev       Network device descriptor
+ * @param[in] dev       Pointer to the display device
  * @param[in] invert    Invert mode (true if invert, false otherwise)
  */
 void disp_dev_set_invert(disp_dev_t *dev, bool invert);
 
+/**
+ * @brief   Enable the backlight pin
+ */
+inline void disp_dev_backlight_on(void)
+{
+    BACKLIGHT_ON;
```

Comment:
Should these be guarded with an ifdef for when the board does not have backlight control pins?

Reply:
There are empty defines added at the beginning of this header, in case the board doesn't provide the defines.

Reply:
Ok!